src/main/webapp/dev/js/common/instructorFeedbacks.js

```diff
@@ -1,7 +1,39 @@
+import {
+    showModalAlert,
+} from './bootboxWrapper';
+
 import {
     ParamsNames,
 } from './const';
 
+const FEEDBACK_SESSION_SESSIONVISIBILITY_DESCRIPTION = 'This option allows you to select when you'
+        + ' want the questions for the feedback session to be visible to users who need to participate.'
+        + ' Users cannot submit their responses until the submission opening time.';
+const FEEDBACK_SESSION_RESULTSVISIBLE_CUSTOM = 'Select this option to enter in a specific date and time when'
+        + ' the feedback session will become visible. Note that you can make a session visible'
+        + ' before it is open for submissions so that users can preview the questions.';
+const FEEDBACK_SESSION_SESSIONVISIBLE_ATOPEN = 'Select this option to have the feedback session become'
+        + ' visible when it is open for submissions.';
+const FEEDBACK_SESSION_RESPONSESVISIBILITY_DESCRIPTION = 'Please select when the responses for the feedback'
+        + ' session will be visible to the designated recipients. You can select the response visibility for'
+        + ' each type of user and question later.';
+const FEEDBACK_SESSION_RESPONSESVISIBLE_CUSTOM = 'Select this option to use a self defined time for when the'
+        + ' responses of the feedback session will be visible to the designated recipients.';
+const FEEDBACK_SESSION_RESPONSESVISIBLE_ONSUBMIT = 'Select this option to have the feedback responses be immediately'
+        + ' visible to others as soon as they are submitted. Note that who can see the responses depends on the feedback'
+        + ' path of the question.';
+const FEEDBACK_SESSION_RESPONSESVISIBLE_LATER = 'Select this option if you intend to manually publish the'
+        + ' responses for this session later on.';
+const FEEDBACK_SESSION_EMAILREMINDERS_DESCRIPTION = 'This setting allows you to automatically send email notifications'
+        + ' to students on important events.';
+const FEEDBACK_SESSION_EMAILREMINDERS_ATOPEN = 'Select this option to automatically send an email to students'
+        + ' to notify them when the feedback session is open for submissions. Emails are sent within 15 minutes'
+        + ' once the session opens.';
+const FEEDBACK_SESSION_EMAILREMINDERS_ATCLOSED = 'Select this option to automatically send an email to students to remind'
+        + ' them to submit their responses 24 hours before the end of the feedback session.';
+const FEEDBACK_SESSION_EMAILREMINDERS_ONPUBLISH = 'Select this option to automatically send an email to students to notify'
+        + ' them when the feedback session results are published.';
+
```

Comment:
It's not a good idea to mix so much HTML in Javascript like this, as it reduces maintainability. Can you try a different approach: Hardcode the HTML in a hidden element in a JSP file, and pass that element as the `messageText` to `showModalAlert` instead?

Reply:
Generating HTML is the job of the JSP files. When we want to change the modal content, that's where we're going to look for the HTML of the modal box.

Btw your code keeps repeating the same pattern for different similar modal. You should abstract that as a JSP tag and pass the appropriate parameters. I'm talking in quite high level language so If you don't understand what I said, I suggest you read up some tutotials on JSP and then we discuss what the JSP file wilk look like first before continuing.